python/lsst/daf/butler/butler.py

```diff
@@ -27,7 +27,8 @@
 import contextlib
 import logging
 
-from .core.utils import doImport, transactional
+from lsst.utils import doImport
```

Comment:
You probably should move the lsst.utils imports before the butler imports.

Reply:
It is already, isn't it?

Reply:
😄 I think there might be one that isn't later on... Sorry.